### ISSUE TITLE
suspect panda_indegree doesn't work

### ISSUE THREAD

**Issue by twangxxx** [netZoo/netZooPy#73]
I suspect the `panda_indegree` doesn't work as [line 361 in panda.py](https://github.com/netZoo/netZooPy/blob/master/netZooPy/panda/panda.py#L361) attribute `'panda_results'` hasn't been defined before.

**Comment by twangxxx** [comment]
And a suggestion from a novice(me).

```
def return_panda_indegree(self):
        '''Return Panda indegree.'''
        #subset_indegree = self.export_panda_results.loc[:,['gene','force']]
        export_panda_results_pd = pd.DataFrame(self.export_panda_results,columns=['tf','gene','motif','force'])
        subset_indegree = export_panda_results_pd.loc[:,['gene','force']]
        subset_indegree['force']=pd.to_numeric(subset_indegree.force)
        self.panda_indegree = subset_indegree.groupby('gene').sum()
        return self.panda_indegree
```

**Comment by marouenbg** [comment]
fixed in #74
